Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -169,6 +169,13 @@ private extension ReceiptRenderer {
 
         return .localizedStringWithFormat(Localization.receiptFromFormat, storeName)
     }
+
+    private var summarySectionTitle: String {
+        guard let orderID = parameters.orderID else {
+            return Localization.summarySectionTitle
+        }
+        return .localizedStringWithFormat(Localization.summarySectionTitleWithOrderFormat, String(orderID))
```

Comment:
I'm not sure `localizedStringWithFormat` is useful here, as its main purpose is formatting numbers according to the user's locale, but we're passing the `orderID` as a `String`, and I'm not sure we wanted that formatted with a thousands separator anyway. I think `String(format:...)` would be better here.

![Screen Shot 2021-09-30 at 07 53 32](https://user-images.githubusercontent.com/8739/135395434-9f6b205d-7ef9-44d6-84f9-0099c67fb7a1.png)